configuration/topology-proxy-follower.json

```diff
@@ -1,14 +1,9 @@
-[
-    { "nodeId": 0
-    , "nodeAddress":
-      { "addr": "127.0.0.1"
-      , "port": 7776
-      }
-    , "producers":
-      [ { "addr": "18.185.45.45"
-        , "port": 3001
-        , "valency": 1
-        }
-      ]
+{
+  "rProducers": [
```

Comment:
"rProducers"?

Reply:
Meant to be short for real producers. Any naming suggestions?

Reply:
Changed to "Producers"